Dockerfile

```diff
@@ -11,8 +11,11 @@ ARG TAGS=""
 COPY --from=builder /go/src/github.com/openshift/machine-config-operator/instroot.tar /tmp/instroot.tar
 RUN cd / && tar xf /tmp/instroot.tar && rm -f /tmp/instroot.tar
 COPY install /manifests
-RUN if [[ "${TAGS}" == "fcos" ]]; then sed -i 's/rhel-coreos-8/fedora-coreos/g' /manifests/*; \
-    elif [[ "${TAGS}" == "scos" ]]; then sed -i 's/rhel-coreos-8/centos-stream-coreos-9/g' /manifests/*; fi && \
+# For now, prevent this mangling from acting on the extensions container because while we aren't
+# shipping fedora-coreos-extensions, we are shipping rhel-coreos-8-extensions, and if it mangles that in image-references, it will break 
+# the okd-images test 
+RUN if [[ "${TAGS}" == "fcos" ]]; then sed -i '/rhel-coreos-8-/! s/rhel-coreos-8/fedora-coreos/g' /manifests/*; \
+    elif [[ "${TAGS}" == "scos" ]]; then sed -i '/rhel-coreos-8-/! s/rhel-coreos-8/centos-stream-coreos-9/g' /manifests/*; fi && \
```

Comment:
We will have an extensions image for `scos`, so this needs to stay as it was.
We also have to make sure the `rhel-coreos-8-extensions` image ref is either removed or commented out in the `fcos` case.

Reply:
Alright, thanks, I put SCOS back, how about this for fcos? :smile:: 
https://github.com/openshift/machine-config-operator/blob/7483303e6a909238820a891b45b5ea84eac93834/Dockerfile#L15-L22

Reply:
Thank you! The `0000_80_machine-config-operator_05_osimageurl.yaml` ends up looking like this, which is fine IMO: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_machine-config-operator/3317/pull-ci-openshift-machine-config-operator-master-okd-images/1569582264726065152/artifacts/release/artifacts/release-payload-latest/0000_80_machine-config-operator_05_osimageurl.yaml